sympy/polys/galoistools.py

```diff
@@ -1877,6 +1877,7 @@ def gf_edf_zassenhaus(f, n, p, K):
 
     .. [1] [Gathen99]_
     .. [2] [Geddes92]_
+    .. [3] [Cohen93]_
```

Comment:
So which algorithm is implemented now? It would be good to add a note clarifying which of these references is the one that describes the implemented algorithm.

Reply:
I see your point. A more direct reference could be made.